graphdatascience/query_runner/arrow_query_runner.py

```diff
@@ -272,8 +289,15 @@ def _run_arrow_property_get(self, graph_name: str, procedure_name: str, configur
             "procedure_name": procedure_name,
             "configuration": configuration,
         }
-        ticket = flight.Ticket(json.dumps(payload).encode("utf-8"))
 
+        if self._arrow_version == ArrowVersion.V1:
+            payload = {
+                "name": "GET_MESSAGE",
```

Comment:
you could put `GET_MESSAGE` / `PUT_MESSAGE` also behind constants as they are tied to ArrowVersion.V1

Reply:
but they are also only used once 🤷

Reply:
I will update those to the final version once I updated the server side